.stylelintrc

```diff
@@ -0,0 +1,147 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": [ "always", {
+      except: [
+        "blockless-after-same-name-blockless",
+        "first-nested",
```

Comment:
Thats a good point, but this rule isn't for just `@extends` or `@includes`, but for anything that has an `@` before. The nested option is there because we will still support .less files on pretty much all of our current front-end projects. The rule only applies to blockless with `@` and the exception is there so that you don't need to put an empty line before the first `@` property of the block. Your example still fails because of other rules that are in place.

<img width="698" alt="style_guide_css_ ___src_servers_crowdtap" src="https://cloud.githubusercontent.com/assets/171380/21202602/07bf53dc-c21d-11e6-982c-c6f50ae1c29a.png">